app/controllers/api/v1/featured_tags_controller.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class Api::V1::FeaturedTagsController < Api::BaseController
+  before_action -> { doorkeeper_authorize! :read, :'read:featured_tags' }, only: [:index, :show]
+  before_action -> { doorkeeper_authorize! :write, :'write:featured_tags' }, except: [:index, :show]
```

Comment:
Same thing here, `read:accounts`, `write:accounts`

Reply:
Change to `read:accounts`, `write:accounts`